test/action-test.ts

```diff
@@ -93,6 +93,30 @@ test('actions can be passed and invoked with additional arguments', function(ass
   assert.deepEqual(passed, [1, 2, 3, 4, 5, 6, fakeEvent]);
 });
 
+test('action helper invoked on a tag with no arguments works', function(assert) {
+  assert.expect(2);
+  let fakeEvent: any = {
+    type: 'click'
+  };
+  let parentComponent: ParentComponent;
+  let passed = false;
+  class ParentComponent extends TestComponent {
+    debugName = 'ParentComponent';
+    doesExist() {
+      passed = true;
+      assert.strictEqual(this, parentComponent, 'function context is preserved');
```

Comment:
The test is failing because you are asserting that the function is bound to `parentComponent`, an uninitialized variable. Also the number of expected assertions is wrong.